packages/back-end/src/api/projects/deleteProject.ts

```diff
@@ -0,0 +1,31 @@
+import { DeleteProjectResponse } from "../../../types/openapi";
+import { createApiRequestHandler } from "../../util/handler";
+import { deleteProjectValidator } from "../../validators/openapi";
+import { auditDetailsDelete } from "../../services/audit";
+
+export const deleteProject = createApiRequestHandler(deleteProjectValidator)(
+  async (req): Promise<DeleteProjectResponse> => {
+    let id = req.params.id;
+    // Add `prj__` prefix if it doesn't exist
+    if (!id.startsWith("prj__")) {
+      id = `prj__${id}`;
+    }
```

Comment:
Remove this.  I'm assuming this was copied from fact metrics, but that's a very unique case where the prefix is meaningful.

Reply:
Well glad I hadn't included it in the tests. I though this might be superfluous..